### PR TITLE
Hide search until SOLR whitelist comes through

### PR DESCRIPTION
**Pull request by sdreher**
scheduled for Monday evening

### CODE REVIEW

**Comment by thraxil** [comment]
good use-case for a feature flag...

**Comment by sdreher** [comment]
Excellent point. I'll redo this with a flag

File: footprints/templates/base.html [hunk 1]
```diff
@@ -67,7 +67,9 @@
                                 {% endif %}
                             {% endif %}
                             <li><a href="/browse/footprints/">Browse</a></li>
-                            <li><a href="/search/">Search</a></li>
+                            {% switch "enable_search" %}
```
Comment (thraxil) [inline footprints/templates/base.html]: why a switch rather than a flag?

**Comment by sdreher** [comment]
This is on/off functionality. No need for anything more complicated.

**Comment by thraxil** [comment]
But a switch means that it's on or off for everyone. A flag leaves open the ability to eg, enable it only for one user so you can test while keeping it off for everyone else.

**Pull request merged**